demisto_sdk/commands/validate/file_validator.py

```diff
@@ -334,26 +340,27 @@ def validate_modified_files(self, modified_files):  # noqa: C901
                     self._is_valid = False
 
             # incident fields and indicator fields are using the same scheme.
-            elif checked_type(file_path, JSON_INDICATOR_AND_INCIDENT_FIELDS):
+            elif checked_type(file_path, JSON_INDICATOR_AND_INCIDENT_FIELDS) or \
+                    file_type in ('incidentfield', 'indicatorfield'):
                 incident_field_validator = IncidentFieldValidator(structure_validator)
-                if not incident_field_validator.is_valid_file():
+                if not incident_field_validator.is_valid_file(validate_rn=False if file_type else True):
```

Comment:
are we sure that both type of fields has the same validations? looks confusing

Reply:
I know, but yes.